middleware/auth/jwt/jwt.go

```diff
@@ -104,27 +104,23 @@ func Server(keyFunc jwt.Keyfunc, opts ...Option) middleware.Middleware {
 					tokenInfo, err = jwt.Parse(jwtToken, keyFunc)
 				}
 				if err != nil {
-					ve, ok := err.(*jwt.ValidationError)
```

Comment:
Why do you want to change the logic here?

Reply:
Sorry, this logic has been resolved  on my next commit